core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -379,9 +476,9 @@ object ConsumerGroupCommand {
   }
 
   class ConsumerGroupCommandOptions(args: Array[String]) {
-    val ZkConnectDoc = "REQUIRED (only when using old consumer): The connection string for the zookeeper connection in the form host:port. " +
+    val ZkConnectDoc = "REQUIRED (unless new-consumer is used): The connection string for the zookeeper connection in the form host:port. " +
```

Comment:
To be honest, it's been so long that I don't recall why I made these changes. But when I try the current command, I can't tell what the meaning of "new consumer being the default" is. It seems to me that we need to either provide `--zookeeper` or `--bootstrap-server`; it's not like if we use `--new-consumer` we don't have to provide `--bootstrap-server`. The only restriction around `--new-consumer` seems to be that it cannot be used along with `--zookeeper`. The descriptions in parenthesis are not very clear to me. But it might be just me. Your thoughts?